Source/Core/Common/NandPaths.cpp

```diff
@@ -54,6 +56,21 @@ void ShutdownWiiRoot()
 {
   if (!s_temp_wii_root.empty())
   {
+    u64 tmd_title_id = Movie::GetTitleId();
```

Comment:
Seems like I was wrong – the title ID in Movie gets set regardless of whether a movie is running: https://github.com/dolphin-emu/dolphin/blob/a3d2dead76f9fe46163a20f6b7cc6ac7b2b66f29/Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_es.cpp#L1176

But this raises the question of why Movie is keeping track of the title ID when it's not just used for movies.